### ISSUE TITLE
Mojolicious::Routes::Route::route is DEPRECATED

### ISSUE THREAD

**Issue by eserte** [jhthorsen/toadfarm#32]
t/app-class.t fails on some of my systems (with newer Mojolicious versions maybe?):
```
#   Failed test 'no warnings'
#   at t/app-class.t line 18.
#     Structures begin differing at:
#          $got->[0] = 'Mojolicious::Routes::Route::route is DEPRECATED in favor of Mojolicious::Routes::Route::any at /home/cpansand/.cpan/build/2021012606/Toadfarm-0.82-1/blib/lib/Toadfarm.pm line 185.
#     '
#     $expected->[0] = Does not exist
# Looks like you failed 1 test of 9.
t/app-class.t ........ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/9 subtests 
```

**Comment by jberger** [comment]
just going to add to this one that route, via, over, and detour have all been removed in  favor of methods listed in https://github.com/mojolicious/mojo/wiki/Upgrading (though I suspect @jhthorsen is aware, just adding for completeness)

**Comment by leejo** [comment]
I have made a start on this in #33

**Comment by jhthorsen** [comment]
Pretty sure @leejo fixed this in #33.
